librz/bin/pdb/tpi.c

```diff
@@ -20,6 +20,48 @@ static bool is_simple_type(int idx) {
 	// return ((value & 0x00000000FFF00) <= 0x700 && (value & 0x00000000000FF) < 0x80);
 }
 
+static int skip_padding(uint8_t *leaf_data, unsigned int *read_bytes, unsigned int len) {
+	unsigned int c = 0;
+	while ((*leaf_data & 0xf0) == 0xf0) {
+		CAN_READ((*read_bytes + c), 1, len);
+		c++;
+		leaf_data++;
+	}
+	(*read_bytes) += (c);
+	return c;
+}
+
+/**
+ * @brief Parses calling convention
```

Comment:
Maybe rename `parse_calling_convention` to `pdb_calling_convention_as_string`? Also description is a bit misleading.